docs/sdk/serverless-signing/quick-start.md

```diff
@@ -7,557 +7,188 @@ import TabItem from '@theme/TabItem';
 
 ## Start Here
 
-Blockchains like Ethereum have smart contracts that let developers encode logic to change that state. As a key management network, Lit provides a method that allows developers to encode logic that dictates signing.
+This guide will walk you through setting up a connection to the Lit network, defining a Lit Action, and executing the Lit Action. The code snippets can be compiled and run to successfully execute the Lit Action, as long as an Ethereum wallet is initialized using the `ETHEREUM_PRIVATE_KEY` environment variable.
 
-Lit Actions are JavaScript functions that can be used read and write data across blockchains, web2 platforms, and the rest of the web3 world. You can use Lit Actions to generate signatures when your specified on or off-chain conditions are met, fetch data from off-chain platforms, and manage permissions for PKPs.
+If you're instead looking to learn more about the concept of Lit Actions, check out our [Overview](./overview).
 
-In the following guide, we will write a simple Lit Action that requests a signature from the Lit nodes and signs the message "Hello World".
+This guide uses Lit's [Datil-dev Network](../../connecting-to-a-lit-network/testnets#datil-dev) which is designed for application developers aiming to get familiar with the Lit SDK. Payment is not required on this network, and therefore the code is less complex. For those wanting to develop using Lit for production-ready applications, the [Datil-test](../../connecting-to-a-lit-network/testnets#datil-test) Network is recommended. More on Lit networks can be found [here](../../network/networks/testnet.md).
```

Comment:
Let's just make sure that we make it clear that datil-test is for pre-production and datil is the production network.

Reply:
Changed to:

```
This guide uses Lit's [Datil-dev Network](../../connecting-to-a-lit-network/testnets#datil-dev) which is designed for application developers aiming to get familiar with the Lit SDK. Payment is not required on this network, and therefore the code is less complex. For those aiming to build production-ready applications, the [Datil-test Network](../../connecting-to-a-lit-network/testnets#datil-test) is recommended. Once ready, these applications can then be deployed on [Datil](../../connecting-to-a-lit-network/mainnets#datil), the Lit production network. To see the available Lit networks, check out [this page](../../connecting-to-a-lit-network/connecting).
```

Reply:
solid